src/mpyl/cli/commands/projects/lint.py

```diff
@@ -73,6 +73,7 @@ def _check_and_load_projects(
 
 
 def _assert_unique_project_names(console: Console, all_projects: list[Project]):
+    console.print("")
```

Comment:
some leftovers from debugging? also on line 101

Reply:
This white line between actions is here on purpose for better readability